env/jenkins-x-platform/values.tmpl.yaml

```diff
@@ -181,6 +181,43 @@ controllerbuild:
   - "--verbose"
 {{- end }}
 
+{{- if hasKey .Requirements.cluster "strictPermissions" }}
```

Comment:
One needs to explicitly check the existence?

Reply:
Would the following work and be easier to read:

```
{{- if and (eq .Requirements.cluster.provider "openshift") (hasKey .Requirements.cluster "strictPermissions") (.Requirements.cluster.strictPermissions) }}
```

Reply:
It evaluates all conditions at the same time sadly, so if the key is not present, the last one will fail.

Reply:
> It evaluates all conditions at the same time sadly, so if the key is not present, the last one will fail.

wau, not what I would have expected. Good to know.